client/extensions/woocommerce/components/dashboard-widget-row/style.scss

```diff
@@ -0,0 +1,14 @@
+@include breakpoint( ">960px" ) {
+	.dashboard-widget-row {
+		display: flex;
+
+		div {
```

Comment:
We should be more specific here. A generic class should be applied to each dashboard widget, we should use that class for the styles here.